package.json

```diff
@@ -16,6 +18,12 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.20.0",
-    "serve": "^13.0.2"
+    "eslint": "^8.11.0",
+    "eslint-config-next": "^12.1.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "^2.6.0",
+    "serve": "^13.0.2",
+    "typescript": "^4.6.2"

```

Comment:
You've added TypeScript but the codebase is JS, is that a mistake

Reply:
Mistake indeed! Thank you. Fixed